nodejs/private/toolchains_repo.bzl

```diff
@@ -63,11 +63,37 @@ PLATFORMS = {
 }
 
 def _impl(repository_ctx):
+    # Expose a concrete toolchain which is the result of Bazel resolving the toolchain
+    # for the execution or target platform.
+    # Workaround for https://github.com/bazelbuild/bazel/issues/14009
+    starlark_content = """# Generated by toolchains_repo.bzl
+
+load("@rules_nodejs//nodejs:toolchain.bzl", "node_toolchain_providers")
+
+def _impl(ctx):
```

Comment:
This re-declaration of `_impl` isn't really a re-declaration, because it's in a string constant, but it's very confusing.

I know we do it a lot in Bazel, but naming a rule implementation `_impl` is just asking for trouble and isn't very future-proof. I'd recommend using `_resolved_toolchain_impl` here (and fixing the previous use of `_impl` to something more specific, also).